testdata/schema/sqlserver.sql

```diff
@@ -18,6 +21,15 @@ CREATE TABLE tags (
 	,created_at DATETIME NOT NULL
 	,updated_at DATETIME NOT NULL
 );
+GO
+
+CREATE SCHEMA x AUTHORIZATION dbo;
+GO
+
+CREATE TABLE x.empty_table_without_fixtures (
+  id INT IDENTITY PRIMARY KEY
```

Comment:
Let's keep indentation with tabs for now, to be consistent with the rest of the file